app/components/satis/menu/component.html.slim

```diff
@@ -1,9 +1,9 @@
-div data-controller="satis-menu" data-action="mouseover->satis-menu#show mouseleave->satis-menu#hide"
+div data-controller="satis-menu" data-action=menu.event
   - if content
     = content
   - else
     button.inline-flex.items-center.justify-center.h-8.w-8.rounded-full.focus:outline-none.focus:ring-2.focus:ring-offset-2.focus:ring-primary-500.dark:text-gray-500 aria-expanded="false" aria-haspopup="true" type="button"
-      span.font-semibold.flex-1 class="#{menu.items.present? ? '' : 'text-gray-200'}"
+      span.font-semibold.flex-1 id="filter_icon#{selected_column}" class="#{menu.items.present? ? '' : 'text-gray-200'}"
```

Comment:
What is selected_column in relation to a menu?

Reply:
This is for handling filter icon of selected column, we need to morph only the selected column filter so used this.